projects/ad4630_fmc/common/ad4630_fmc.txt

```diff
@@ -0,0 +1,22 @@
+# ad4630
+
+FMC_pin   FMC_port       Schematic_name     System_top_name     IOSTANDARD  Termination
+
+G6        LA00_P_CC      SCK_FMC            ad463x_spi_sclk     LVCMOS25    #N/A
+G7        LA00_N_CC      CS_FMC             ad463x_spi_cs       LVCMOS25    #N/A
+G9        LA03_P         SDO2_FMC           ad463x_spi_sdi[2]   LVCMOS25    #N/A
+G10       LA03_N         SDO3_FMC           ad463x_spi_sdi[3]   LVCMOS25    #N/A
+H4        CLK0_M2C_P     CLK                ad463x_ext_clk      LVCMOS25    #N/A
+H7        LA02_P         SDO0_FMC           ad463x_spi_sdi[0]   LVCMOS25    #N/A
+H8        LA02_N         SDO1_FMC           ad463x_spi_sdi[1]   LVCMOS25    #N/A
+H10       LA04_P         SDO4_FMC           ad463x_spi_sdi[4]   LVCMOS25    #N/A
+H11       LA04_N         SDO5_FMC           ad463x_spi_sdi[5]   LVCMOS25    #N/A
+D8        LA01_P_CC      CNV_FMC            ad463x_cnv          LVCMOS25    #N/A
+D9        LA01_N_CC      RESET_FMC          ad463x_resetn       LVCMOS25    #N/A
+D11       LA05_P         SDO6_FMC           ad463x_spi_sdi[6]   LVCMOS25    #N/A
+D12       LA05_N         SDO7_FMC           ad463x_spi_sdi[7]   LVCMOS25    #N/A
+D20       LA17_P_CC      SCK_OUT_FMC        ad463x_echo_sclk    LVCMOS25    #N/A
+C11       LA06_N         SDI_FMC            ad463x_spi_sdo      LVCMOS25    #N/A
+C22       LA18_P_CC      BUSY_FMC           ad463x_busy         LVCMOS25    #N/A
+C30       SCL            SCL                iic_scl             LVCMOS33    #N/A
```

Comment:
The SCL and SDA pins are already constrained in zed_system_constr.xdc . Can you please remove them from this file?